src/images/pages/regulation/map.svg

```diff
@@ -3377,7 +3377,7 @@
 			<path class="st55" d="M8.7,93c-0.1,0-0.4,0-0.6,0l-3,1.8l0,0h0.7L8.7,93L8.7,93z"/>
 			<path class="st55" d="M10.8,91c0.1,0,0.3,0,0.6,0l3-1.8l0,0h-0.7L10.8,91C10.8,90.9,10.8,91,10.8,91z"/>
 		</g>
-		<polygon class="st55" points="14.4,91.4 10.4,91.4 10.4,89.1 9.1,89.1 9.1,91.4 5.1,91.4 5.1,92.6 9.1,92.6 9.1,94.8 10.4,94.8 
+		<polygon class="st55" points="14.4,91.4 10.4,91.4 10.4,89.1 9.1,89.1 9.1,91.4 5.1,91.4 5.1,92.6 9.1,92.6 9.1,94.8 10.4,94.8
```

Comment:
I'm confused, what are these changes? are they about alignment? can you revert any change in this file entirely?